src/Avalonia.Controls/Window.cs

```diff
@@ -171,13 +171,13 @@ public class Window : WindowBase, IStyleable, IFocusScope, ILayoutRoot
         /// <summary>
         /// Routed event that can be used for global tracking of window destruction
         /// </summary>
-        public static readonly RoutedEvent WindowClosedEvent =
```

Comment:
This is a breaking change so I'm going to remove the `backport-candidate` label.